changelogs/unreleased/gh-10596-connpool-filter-support-alive-filtering.md

```diff
@@ -0,0 +1,4 @@
+## feature/connpool
+
+- Now `connpool.filter()` supports filtering only alive instances by
+  specifying `'alive'` mode (gh-10596).
```

Comment:
```suggestion
- Now `connpool.filter()` supports a new filtering mode, `'alive'`, for filtering only 
  alive instances (gh-10596).
```

Reply:
Thank you. I've had to change the whole message due to the PR update. Please, review it one more time.